### ISSUE TITLE
"Complete task" not including all the necessary information when request id is not in accepted mode

### ISSUE THREAD

**Issue by StergiosSozos** [anitab-org/mentorship-backend#537]
## Description
As a user,when i try to complete a task I need to know what I did wrong and why what I expected did not happen.

I observed that when a request id is not accepted and a user tries to complete a task (which does not exist, as a task cannot be created) the message is about the wrong task, but it should show the initial problem too, that the relationship is not accepted.

**To Reproduce**
1. Send a relationship request from user1 to a user2
2. Do not accept that request from user2
3. Try to complete a task (the number will be random as a task cannot be created)
4. See the error message, about the task not existing, but not about the relationship not having been accepted

**Expected behavior**
The message should inform the user trying to complete the task that the relationship is not accepted too.

**Screenshots**
![image](https://user-images.githubusercontent.com/44146773/79069036-cdbed200-7cd3-11ea-9301-03b03a8ba965.png)

**Comment by StergiosSozos** [comment]
@isabelcosta  Can i work on this issue?

**Comment by StergiosSozos** [comment]
@isabelcosta

**Comment by StergiosSozos** [comment]
@isabelcosta can i be assigned please?

**Comment by isabelcosta** [comment]
Hi @StergiosSozos sure I approve this. So if I understand correctly, your idea here is to first respond with the highest level of what went wrong, and in this case was that the request is not in the accepted state. Is this correct?

**Comment by StergiosSozos** [comment]
Exactly! Now, we inform the user that there is not task, but there is no point in this as the user cannot create a task, he first needs to have an accpeted relationship. I already worked on that so I will create a PR when it's ready! Sorry for working before getting assigned.

**Comment by b-thebest** [comment]
Can I work on this issue

**Comment by RiddhiAthreya** [comment]
@vj-codes can I work on this?

**Comment by isabelcosta** [comment]
@RiddhiAthreya if you're still interested, ping me and I'll assign this to you ;)
